[docs] ACL reset procedure (#5334)
hashicorp/consul: website/source/docs/guides/acl.html.md

All the values should be as expected. Particularly, if `TaggedAddresses` is `null` it is likely we have not configured ACLs correctly. A good place to start debugging is reviewing the Consul logs on all the servers.

If you encounter issues that are unresolvable, or misplace the bootstrap token, you can reset the ACL system by updating the index. First re-run the bootstrap command to get the index number.

```
$ consul acl bootstrap

> Error bootstrapping: Unexpected response code: 500 (ACL bootstrap already done (reset index: 13))

Reviewer: ```suggestion
Failed ACL bootstrapping: Unexpected response code: 403 (Permission denied: ACL bootstrap no longer allowed (reset index: 13))
```

This is the actual error.

```

> Then update the index with the following command.

Reviewer: ```suggestion
Then write the reset index into the bootstrap reset file: (here the reset index is 13)
```

Maybe something more like this would be better. Basically what you are doing is writing the reset index you got from the first call into the bootstrap reset file to indicate to Consul that it should allow bootstrapping again.

Author: LGTM. I like that you were more specific, no room for questions :)

```

> $ echo 13 >> /<data-directory>/server/acl-bootstrap-reset

Reviewer: ```suggestion
$ echo 13 >> <data-directory>/acl-bootstrap-reset
```

Reviewer 2: the <data-directory> is means the -data-dir what i set? i try, but it's doesn't work

Reviewer 2: the data-directory is means the -data-dir what i set? i try, but it's doesn't work
